2_process/src/combine_NLCD_PRMS.R

```diff
@@ -0,0 +1,58 @@
+###-----
+## Subset Land to AOI and join with area and PRMS ids
+AOI_LC_w_area <- function(NLCD_LC_df, area_att, aoi_comids_df){
```

Comment:
first stand-alone function takes the LC dataset and subsets + joins with PRMS ids from the drb_comids_df